source/uwp/Renderer/lib/ActionHelpers.cpp

```diff
@@ -689,7 +689,7 @@ namespace AdaptiveCards::Rendering::Uwp::ActionHelpers
         auto showCardActionConfig = actionsConfig.ShowCard();
         auto showCardActionMode = showCardActionConfig.ActionMode();
         // ActionMode enum exists both in Rendering in ObjectModel namespaces. When the time permits, fix it.
-        if (showCardActionMode == winrt::render_xaml::ActionMode::Inline)
+        if (showCardActionMode == winrt::winrt_render_xaml::ActionMode::Inline)
```

Comment:
`winrt::winrt_render_xaml` - twice winrt?:)
why don't we just use the fully qualified namespace here if it's used so infrequently :)

Reply:
The fully qualified namespace contains "uwp", and I'm trying to remove those so there are fewer #ifdefs in the future. I don't love the double winrt but I think it's the most accurate and understandable alias here, and it only needs to occur once (the other instances of winrt_render_xaml don't need the winrt:: prefix). If we ever get around to the todo in the comment above, we can remove this entirely.

Reply:
I'm not sure what that gets us? I think we're going to need to do some weird things with the Uwp namespace so I'm really trying to keep it contained.